src/expression.ts

```diff
@@ -156,6 +156,28 @@ export class BinaryExpression extends Expression {
     }
 }
 
+/** it uses to specified a wasm function call in compile time,
+ * sometimes the wasm function maybe is not generated from a ts function
+ */
+export class CallWASMFunction extends Expression {
```

Comment:
Also, we shouldn't introduce such concept in frontend, this is a special design in binaryen backend.